app/containers/HomeContainer/tests/index.test.js

```diff
@@ -7,11 +7,17 @@
 import React from 'react';
 import { timeout, renderProvider } from '@utils/testUtils';
 import { fireEvent } from '@testing-library/dom';
-import { HomeContainerTest as HomeContainer } from '../index';
+import getIntl from '@utils/getIntl';
+import { setIntl, translate } from '@app/components/IntlGlobalProvider';
+import { HomeContainerTest as HomeContainer, mapDispatchToProps } from '../index';
 
 describe('<HomeContainer /> tests', () => {
   let submitSpy;
 
+  beforeAll(() => {
```

Comment:
we don't really need to do this for component tests

Reply:
Fixed